recipe/meta.yaml

```diff
@@ -6,38 +6,54 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dbt-snowflake-{{ version }}.tar.gz
-  sha256: 959943cdb0bcedc67203f0205a9ae21fba13f4c95ddf7089848a77b8c8a8d5f1
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dbt-snowflake-{{ version }}.tar.gz
+    sha256: 959943cdb0bcedc67203f0205a9ae21fba13f4c95ddf7089848a77b8c8a8d5f1
+
+  - url: https://github.com/dbt-labs/dbt-snowflake/archive/refs/tags/v{{ version }}.zip
+    sha256: 01e450dd7b6737a96e40d50bca25aa02ea2f69f6f274b43f7c80ea7cccb737dc
+    folder: dbt-snowflake-git
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
   number: 0
+  # Skip s390x because we don't have agate on s390x.
+  skip: true  # [py<38 or s390x]
 
 requirements:
   host:
     - pip
-    - python >=3.8
+    - python
+    - wheel
+    - setuptools
   run:
-    # via Grayskull
-    - python >=3.8
+    - python
     - dbt-core >=1.7.0,<1.8.dev0
     - snowflake-connector-python >=3.0,<4.dev0
+    # installed via dbt-core but referenced directly; don't pin to avoid version conflicts with dbt-core
     - agate
 
 test:
+  source_files:
+    - dbt-snowflake-git/tests
   imports:
     - dbt.adapters.snowflake
   commands:
-    - pip check
+    # Skip pip check because dbt-core depends on protobuf 4 but we modified the dep to use 3 instead.
+    # - pip check
+    - pytest -v dbt-snowflake-git/tests/unit
   requires:
     - pip
+    - pytest
 
 about:
   home: https://github.com/dbt-labs/dbt-snowflake
-  summary: The Snowflake adapter plugin for dbt
+  summary: The Snowflake adapter plugin for dbt.
+  description: dbt-snowflake contains all of the code enabling dbt to work with Snowflake.
   license: Apache-2.0
+  license_family: Apache
   license_file: LICENSE.md
+  dev_url: https://github.com/dbt-labs/dbt-snowflake
+  doc_url: https://docs.getdbt.com/docs/core/connect-data-platform/snowflake-setup
 
 extra:
```

Comment:
```suggestion
extra:
   skip_lints:
        - missing_pip_check
```

Reply:
The reason to not add the skip is because we will forget to remove it once the problem with protobuf is fixed. So I much prefer to have a lint warning than have an error in prod because of an outdated silenced warning.

Reply:
I would hope someone would notice it needs to removed when the time comes, but I see your point as well.

Reply:
The problem is that it's at the bottom of the file, without any context. Skipping a linting rule should be done inline or very close to the line in question, not at the bottom of a file. That's a flaw of our linter unfortunately.

Reply:
Maybe add a comment stating the skip needs to be removed once `pip check` is reinstated? Your call either way.

Reply:
I'm a sucker for those green check marks!